templates/include/telemetry/syncMigration.1.schema.json

```
@@ -0,0 +1,19 @@
+{
+    "type": "object",
+    "required": ["type"],
+    "properties": {
+        "type": { "type": "string" }
+    },
+    "anyOf": [
+        {
+            "properties": {
+                "type": { "const": "webext-storage" },
+                "entries": { "type": "integer" },
```

You can also add more validation (just in case that is something you might want), e.g.:
```suggestion
                "entries": { "type": "integer", "minimum": 0 },
```